tests/functional/resolvers.ts

```diff
@@ -224,6 +246,12 @@ describe("Resolvers", () => {
         type => type.name === "SampleObject",
       ) as IntrospectionObjectType;
       argMethodField = sampleObjectType.fields.find(field => field.name === "argMethodField")!;
+      sampleInputType = schemaIntrospection.types.find(
```

Comment:
This variables are not used globally in this test suite - move them do the used block:
```ts
describe("Input object", () => {
  let sampleInputType: IntrospectionInputObjectType;
  let sampleInputChildType: IntrospectionInputObjectType;

  beforeAll(() => {
    sampleInputType = schemaIntrospection.types.find(
      field => field.name === "SampleInput",
    )! as IntrospectionInputObjectType;
    sampleInputChildType = schemaIntrospection.types.find(
      field => field.name === "SampleInputChild",
    )! as IntrospectionInputObjectType;
  });
```